test/Renci.SshNet.IntegrationTests/SshClientTests.cs

```diff
@@ -91,29 +91,6 @@ public void Send_InputStream_to_Command_OneByteAtATime()
             }
         }
 
-        [TestMethod]
-        public void Send_InputStream_to_Command_InputStreamNotInUsingBlock_StillWorks()
```

Comment:
I deleted this test because now it hangs. It was nice that it didn't hang before, but I don't think that behaviour is compatible with a TAP implementation. Besides, if you are sending a stream of data to `cat` without telling it when you have finished, and you aren't using any cancellation or timeout feature, then it is surely only logical that the command would keep executing forever.